corehq/apps/userreports/reports/factory.py

```diff
@@ -41,7 +41,7 @@ class ReportColumnFactory(object):
     }
 
     @classmethod
-    def from_spec(cls, spec, is_static, domain):
```

Comment:
This makes me a little nervous, because `domain` isn't really optional.  If some real code path neglects to pass it in, what's the failure mode?  I'd guess either a hard exception or unwarranted access to the feature, though I'm not certain which.

Reply:
If you don't pass domain you don't get access (unless you're in a unit test), because calling `enabled` on a toggle with `None` returns false.